packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,14 +1,22 @@
 const request = require('../exporters/common/request')
 let seqId = 0
+
+function getPayload (payload) {
+  if (Array.isArray(payload)) {
+    return payload
```

Comment:
Should array elements also be trimmed?

Reply:
At the moment, there is only one use case where we need to send an array and it doesn't contain elements to be trimmed

Reply:
If the array is a list of regular payloads then I would expect it goes through the same code path even if not needed right now. Otherwise it would mean it's a completely different format and then a comment might be in order to explain that.

Reply:
In theory, it is not a list of regular payloads. It is a single payload of array type that contains log entries.
You can check the spec [here](https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/SchemaDocumentation/Schemas/logs.md)

But, I can trim elements inside array.

Reply:
That spec is not publicly available. Let's copy the important bits as a comment in code.

Reply:
ok, done